Wrapping/Python/vtk/qt4/QVTKRenderWindowInteractor.py

```diff
@@ -218,11 +217,27 @@ def Finalize(self):
         self._RenderWindow.Finalize()
 
     def CreateTimer(self, obj, evt):
-        self._Timer.start(10)
+        timer_id = self.GetTimerEventId()
+        timer_duration = self.GetTimerDuration(timer_id)
+        timer_type = self.GetTimerEventType()
+
+        t = QtCore.QTimer()
+        t.connect(t, QtCore.SIGNAL("timeout()"), self.TimerEvent)
+        if timer_type == self.OneShotTimer:
+            t.setSingleShot(True)
```

Comment:
What @dlonie said. By default, SingleShot is False, so the timer will be repeating.